src/pages/workspace/expensifyCard/WorkspaceEditCardLimitTypePage.tsx

```diff
@@ -0,0 +1,160 @@
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import Navigation from '@navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+
+// TODO: remove when Onyx data is available
+const mockedCard = {
+    accountID: 885646,
+    availableSpend: 1000,
+    nameValuePairs: {
+        cardTitle: 'Test 1',
+        isVirtual: true,
+        limit: 2000,
+        limitType: CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY,
+    },
+    lastFourPAN: '1234',
+};
+
+type WorkspaceEditCardLimitTypePageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.EXPENSIFY_CARD_LIMIT_TYPE>;
+
+function WorkspaceEditCardLimitTypePage({route}: WorkspaceEditCardLimitTypePageProps) {
+    const {policyID, cardID} = route.params;
+
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${policyID}_${CONST.EXPENSIFY_CARD.BANK}`);
+    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`);
+
+    const card = cardsList?.[cardID] ?? mockedCard;
+    const areApprovalsConfigured = !isEmptyObject(policy?.approver) && policy?.approvalMode !== CONST.POLICY.APPROVAL_MODE.OPTIONAL;
+    const defaultLimitType = areApprovalsConfigured ? CONST.EXPENSIFY_CARD.LIMIT_TYPES.SMART : CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY;
+    const initialLimitType = card.nameValuePairs?.limitType ?? defaultLimitType;
+    const promptTranslationKey =
+        initialLimitType === CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY
+            ? 'workspace.expensifyCard.changeCardMonthlyLimitTypeWarning'
+            : 'workspace.expensifyCard.changeCardSmartLimitTypeWarning';
+
+    const [typeSelected, setTypeSelected] = useState(initialLimitType);
+    const [isConfirmModalVisible, setIsConfirmModalVisible] = useState(false);
+
+    const updateCardLimitType = () => {
+        // TODO: add API call when it's supported https://github.com/Expensify/Expensify/issues/407833
+    };
+
+    const submit = () => {
+        // TODO: update the condition of showing confirm warning when requirements are known
+        const shouldShowConfirmModal = true;
+
+        if (shouldShowConfirmModal) {
+            setIsConfirmModalVisible(true);
+        } else {
+            updateCardLimitType();
+        }
+    };
+
+    const data = useMemo(() => {
+        const options = [];
+        // TODO: update the condition of showing the fixed option when requirements are known
```

Comment:
Just as a heads up, we are planning to introduce a new API endpoint called `OpenWorkspaceEditCardLimitTypePage` that will calculate and fetch the `totalSpend` and the `unapprovedSpend` for a given cardID so we can accurately calculate whether updating the limit type for a card will start to decline transactions for all [these cases](https://expensify.slack.com/archives/C036QM0SLJK/p1721311429909719?thread_ts=1721311425.532449&cid=C036QM0SLJK).


https://github.com/Expensify/Expensify/issues/413936